simulator/src/embedListeners.js

```diff
@@ -1,205 +1,204 @@
-        if (e.keyCode == 16) {
-            simulationArea.shiftDown = false;
-        }
-        if (e.key == 'Meta' || e.key == 'Control') {
-            simulationArea.controlDown = false;
-        }
-    });
-
-    document.getElementById('simulationArea').addEventListener('mousedown', (e) => {
-        errorDetectedSet(false);
-        updateSimulationSet(true);
-        updatePositionSet(true);
-        updateCanvasSet(true);
-
-        simulationArea.lastSelected = undefined;
-        simulationArea.selected = false;
-        simulationArea.hover = undefined;
-        var rect = simulationArea.canvas.getBoundingClientRect();
-        simulationArea.mouseDownRawX = (e.clientX - rect.left) * DPR;
-        simulationArea.mouseDownRawY = (e.clientY - rect.top) * DPR;
-        simulationArea.mouseDownX = Math.round(((simulationArea.mouseDownRawX - globalScope.ox) / globalScope.scale) / unit) * unit;
-        simulationArea.mouseDownY = Math.round(((simulationArea.mouseDownRawY - globalScope.oy) / globalScope.scale) / unit) * unit;
-        simulationArea.mouseDown = true;
-        simulationArea.oldx = globalScope.ox;
-        simulationArea.oldy = globalScope.oy;
-
-
-        e.preventDefault();
-        scheduleUpdate(1);
-    });
-
-    document.getElementById('simulationArea').addEventListener('mousemove', () => {
-        var ele = document.getElementById('elementName');
-        if (globalScope && simulationArea && simulationArea.objectList) {
-            var { objectList } = simulationArea;
-            objectList = objectList.filter((val) => val !== 'wires');
-
-            for (var i = 0; i < objectList.length; i++) {
-                for (var j = 0; j < globalScope[objectList[i]].length; j++) {
-                    if (globalScope[objectList[i]][j].isHover()) {
-                        ele.style.display = 'block';
-                        if (objectList[i] === 'SubCircuit') {
-                            ele.innerHTML = `Subcircuit: ${globalScope.SubCircuit[j].data.name}`;
-                        } else {
-                            ele.innerHTML = `CircuitElement: ${objectList[i]}`;
-                        }
-                        return;
-                    }
-                }
+export default function startListeners () {
+  window.addEventListener('keyup', (e) => {
+    scheduleUpdate(1);
+    if (e.keyCode == 16) {
+      simulationArea.shiftDown = false;
+    }
+    if (e.key == 'Meta' || e.key == 'Control') {
+      simulationArea.controlDown = false;
+    }
+  });
+
+  document.getElementById('simulationArea').addEventListener('mousedown', (e) => {
+    errorDetectedSet(false);
+    updateSimulationSet(true);
+    updatePositionSet(true);
+    updateCanvasSet(true);
+
+    simulationArea.lastSelected = undefined;
+    simulationArea.selected = false;
+    simulationArea.hover = undefined;
+    const rect = simulationArea.canvas.getBoundingClientRect();
+    simulationArea.DownRawX = (e.clientX - rect.left) * DPR;
```

Comment:
change DownRawX to TouchDownX or something similar. Adding "touch" prefix in every variable will be easier to understand in the future.

Reply:
@ShreyaPrasad1209 do you mean `TouchMouseDownX` like this